### DISCUSSION TITLE
Copilot not working - Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by jackiekazil** [community/community#47886] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

```
[INFO] [auth] [2023-02-18T16:56:56.343Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-02-18T16:56:56.345Z] GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found."
```

Suggestions on how to troubleshoot?

**Comment by davecheney** [comment, marked as answer]
@jackiekazil it looks like you don't have a Copilot subscription. https://docs.github.com/en/copilot/quickstart#signing-up-for-github-copilot-for-your-personal-account

**Comment by jackzhid** [comment]
GitHub Copilot 1.77.2.0 (v1.77.2.0@570999d7d)
GitHub Copilot Agent v1.77.0
[DEBUG] [agent] [2023-03-28T18:14:26.358Z] Agent service starting
[DEBUG] [agent] [2023-03-28T18:14:26.568Z] Telemetry initialized
[INFO] [auth] [2023-03-28T18:14:38.534Z] Invalid copilot token: missing token: 403 
[INFO] [auth] [2023-03-28T18:14:38.938Z] Invalid copilot token: missing token: 403 
[INFO] [auth] [2023-03-28T18:14:39.314Z] Invalid copilot token: missing token: 403 
You don't have access to GitHub Copilot.

**Reply by davecheney** [reply]
There is no copilot subscription for the login @jackiekazil

**Reply by jackiekazil** [reply]
I think u tagged the wrong person.

**Reply by georgiyAppolonov** [reply]
Same problem

**Comment by Sai-Akula** [comment]
@davecheney 
could you help me on my issue 
Invalid copilot token: missing token: 403 
You don't have access to GitHub Copilot.
![image](https://user-images.githubusercontent.com/130383550/231240135-fad4c262-cdfe-4cc5-833d-9f8191bb05f7.png)

**Comment by jackiekazil** [comment]
My issue has been resolved, so I am going to close this discussion.

**Comment by iotarchitect** [comment]
I'm getting this on Visual Studio Enterprise. Isn't Copilot included with that???

**Comment by jeffreyski1** [comment]
github copilot still giving this error...
